hazelcast/src/main/java/com/hazelcast/config/XmlConfigBuilder.java

```diff
@@ -500,6 +480,46 @@ private void handleWanReplication(final Node node) throws Exception {
         config.addWanReplicationConfig(wanReplicationConfig);
     }
 
+    private void handleWanTargetConfig(WanReplicationConfig wanReplicationConfig,
+                                       WanTargetClusterConfig wanTarget, Node targetChild) {
+        final String targetChildName = cleanNodeName(targetChild);
+        if ("replication-impl".equals(targetChildName)) {
```

Comment:
I think that is generic problem with all configs that accept fully qualified class names... @bilalyasar , can you please create an issue for that?